SPECS/python-requests-mock/python-requests-mock.spec

```diff
@@ -1,169 +1,94 @@
-# Disable tests on epel8 - dependencies dont exist.
-%bcond_with tests
-%else
-%bcond_without tests
-%endif
-
-Name:           python-requests-mock
-Version:        1.7.0
-Release:        3%{?dist}
+%global pypi_name requests-mock
+Name:           python-%{pypi_name}
+Version:        1.8.0
+Release:        6%{?dist}
 Summary:        A requests mocking tool for python
-
 License:        ASL 2.0
+Vendor:         Mariner
+Distribution:   Microsoft Corporation
 URL:            https://requests-mock.readthedocs.io/
-Source0:        https://pypi.io/packages/source/r/requests-mock/requests-mock-%{version}.tar.gz#/python-requests-mock-%{version}.tar.gz
-
+Source0:        https://pypi.io/packages/source/r/requests-mock/requests-mock-%{version}.tar.gz#/%{pypi_name}-%{version}.tar.gz
 Patch0:         0002-Use-system-urllib3-package.patch
 Patch1:         0003-Allow-skipping-purl-tests-if-it-is-not-present.patch
-
 BuildArch:      noarch
 
 %description
 requests-mock provides a simple way to do HTTP mocking at the
 python-requests layer.
 
-%if %{with python2}
-%package -n python2-requests-mock
+%package -n python3-%{pypi_name}
 Summary:        A requests mocking tool for python
-
-Requires:       python2-requests
-Requires:       python2-six
-
-# This {?fedora:2} syntax is because it's python2- on f28+ but just python- on
-# epel still. Hopefully this can be removed when epel is fixed.
-BuildRequires:  python%{?fedora:2}-urllib3
-
-# standard requirements needed for testing
-BuildRequires:  python2-requests
-BuildRequires:  python2-six
-
-BuildRequires:  python2-devel
-BuildRequires:  python2-pbr
-BuildRequires:  python2-setuptools
-
-%if 0%{?rhel} && 0%{?rhel} <= 6
-BuildRequires:  python-discover
-%endif
-
-%if %{with tests}
-BuildRequires:  python2-mock
-BuildRequires:  python2-pytest
-BuildRequires:  python%{?fedora:2}-fixtures
-BuildRequires:  python%{?fedora:2}-testtools
-%endif
-
-
-%{?python_provide:%python_provide python2-requests-mock}
-
-
-%description -n python2-requests-mock
-requests-mock provides a simple way to do HTTP mocking at the
-python-requests layer.
-%endif
-
-
-%if %{with python3}
-%package -n python%{python3_pkgversion}-requests-mock
-Summary:        A requests mocking tool for python
-
-Requires:       python%{python3_pkgversion}-requests
-Requires:       python%{python3_pkgversion}-six
-
-# standard requirements needed for testing
+BuildRequires:  pyproject-rpm-macros
+BuildRequires:  python%{python3_pkgversion}-pip
+BuildRequires:  python%{python3_pkgversion}-devel
 BuildRequires:  python%{python3_pkgversion}-requests
 BuildRequires:  python%{python3_pkgversion}-six
-BuildRequires:  python%{python3_pkgversion}-urllib3
-
-BuildRequires:  python%{python3_pkgversion}-devel
 BuildRequires:  python%{python3_pkgversion}-pbr
+BuildRequires:  python%{python3_pkgversion}-urllib3
 BuildRequires:  python%{python3_pkgversion}-setuptools
+Requires:       python%{python3_pkgversion}-requests
+Requires:       python%{python3_pkgversion}-six
```

Comment:
Are these the only two requirements this package needs at runtime?

Reply:
I checked this on Fedora 36:
requests_mock-1.8.0.dist-info
METADATA:26:Requires-Dist: requests (<3,>=2.3)
METADATA:27:Requires-Dist: six

Do let me know if this is the right way to confirm.